scripts/cgp_data_loader.py

```diff
@@ -38,6 +38,9 @@ def main(argv):
     parser.add_argument("--staging-bucket", metavar="STAGING_BUCKET", required=False,
                         default=STAGING_BUCKET_DEFAULT,
                         help="Bucket to stage local files for uploading to DSS")
+    parser.add_argument("-l", "--log", dest="logLevel",
+                        choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'],
+                        default="WARNING", help="Set the logging level")
```

Comment:
I think the default should INFO, as in general we want a reasonable amount of informational output as large loads progress, etc.

Reply:
I was planning to add a `--verbose` flag that would set logLevel to INFO and use `verbose` in the example code. I'm happy to change it either way.